aten/src/ATen/native/native_functions.yaml

```diff
@@ -3598,6 +3602,8 @@
 # TensorOptions. Otherwise, an ambiguity error is thrown.
 # See NOTE [ TensorOptions Constructors ].
 - func: to.dtype_layout(Tensor self, *, ScalarType dtype, Layout layout, Device device, bool pin_memory=False, bool non_blocking=False, bool copy=False, MemoryFormat? memory_format=None) -> Tensor
+  #- func: to.dtype_layout(Tensor self, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, bool non_blocking=False, bool copy=False, MemoryFormat? memory_format=None) -> Tensor
+  #use_c10_dispatcher: full
```

Comment:
Would be good to mention why this didn't work

Reply:
I'll just remove those changes from the PR before landing and address them in a separate PR.